game.py

```diff
@@ -20,7 +20,24 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    print("* LETS PLAY SNOWMAN ! *")
+    wrong_guesses_list = []
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if user_input in snowman_word:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+            print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        else: 
+            print (f"The letter {user_input} is not in the word")
+            wrong_guesses_list.append(user_input)
+            print_snowman_graphic(len(wrong_guesses_list))
```

Comment:
👍 Nice job using finding the length of `wrong_guesses_list` and passing that value to `print_snowman_graphic`. Doing so means you don't need to create an unnecessary variable like `wrong_guesses_count` that would need to be manually incremented (which would be a place that a bug could get introduced).